<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectorWidget</class>
 <widget class="QWidget" name="SelectorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>317</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="overviewTab">
      <attribute name="title">
       <string>Overview</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="StrokeFillSelector" name="selector">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QLabel" name="type">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QToolButton" name="typePopup">
           <property name="shortcut">
            <string>Ctrl+S</string>
           </property>
           <property name="popupMode">
            <enum>QToolButton::InstantPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextOnly</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Type:</string>
           </property>
           <property name="buddy">
            <cstring>typeChoice</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QComboBox" name="typeChoice">
           <item>
            <property name="text">
             <string>No color</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Flat color</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Gradient</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bitmap pattern</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Transparency:</string>
           </property>
           <property name="buddy">
            <cstring>transparency</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="transparencySlider">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="transparency">
           <property name="suffix">
            <string> %</string>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>1</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="strokeFillTab">
      <attribute name="title">
       <string/>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Type:</string>
         </property>
         <property name="buddy">
          <cstring>typeChoice</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="typeChoice2">
         <item>
          <property name="text">
           <string>No color</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Flat color</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Gradient</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Bitmap pattern</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="contentsLayout"/>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>1</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StrokeFillSelector</class>
   <extends>QFrame</extends>
   <header>docker/StrokeFillSelector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>typeChoice</tabstop>
  <tabstop>transparencySlider</tabstop>
  <tabstop>transparency</tabstop>
  <tabstop>typePopup</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>transparencySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>transparency</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>transparency</sender>
   <signal>valueChanged(int)</signal>
   <receiver>transparencySlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>typeChoice</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>typeChoice2</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>typeChoice2</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>typeChoice</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
